janusgraph-test/src/main/java/org/janusgraph/blueprints/AbstractJanusGraphProvider.java

```diff
@@ -134,7 +139,11 @@ public void clear(Graph g, final Configuration configuration) throws Exception {
             JanusGraph graph = (JanusGraph) g;
             if (graph.isOpen()) {
                 if (g.tx().isOpen()) g.tx().rollback();
-                g.close();
+                try {
+                    g.close();
+                } catch (IOException | IllegalStateException e) {
+                    logger.warn("Titan graph may not have closed cleanly", e);
```

Comment:
Please create an issue and move on. I do not want to block 3.2.3 support at this point.